src/main/kotlin/oop/Command/ResponsabilityChain/PasswordProcessor.kt

```diff
@@ -0,0 +1,14 @@
+package oop.Command.ResponsabilityChain
+
+class PasswordProcessor(val next : MessageProcessor? = null) : MessageProcessor {
```

Comment:
What do you mean? This is how this pattern works, it is like a linked list from EDA where we had nodes and the last one didn't have successors.

We discussed this in class today because this is a critical point in this pattern. People suggested to always have one last *Processor* which would return the message unaltered.